iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/SubstationAdderImpl.java

```diff
@@ -62,9 +62,7 @@ public Substation add() {
         String id = checkAndGetUniqueId();
         SubstationImpl substation = new SubstationImpl(id, getName(), isFictitious(), country, tso, networkRef);
         if (tags != null) {
-            for (String tag : tags) {
-                substation.addGeographicalTag(tag);
-            }
+            substation.setProperty(Substation.GEOGRAPHICAL_TAGS_KEY, String.join(",", tags));
```

Comment:
Same things that in `Substation` API, I think the `setGeographicalTags(String...)` method should be deprecated.